src/components/Button.test.tsx

```diff
@@ -50,3 +50,29 @@ it('renders DecoyButton', () => {
   const { container } = render(<DecoyButton>DecoyButton</DecoyButton>)
   expect(container.firstChild).toMatchSnapshot()
 })
+
+it('works properly with clicks and touches', () => {
+  const onPress = jest.fn()
+  const { getByText } = render(<Button onPress={onPress}>Test Button</Button>)
+  const button = getByText('Test Button')
+
+  fireEvent.click(button)
+  expect(onPress).toHaveBeenCalledTimes(1)
+
+  fireEvent.pointerDown(button)
+  expect(onPress).toHaveBeenCalledTimes(2)
+
+  // this is the behavior when a proper touch becomes a click
+  fireEvent.pointerDown(button)
+  fireEvent.pointerUp(button)
+  fireEvent.click(button)
+  expect(onPress).toHaveBeenCalledTimes(3)
+
+  // this is the behavior on a bad touch, followed by a separate click
+  // so this ends up calling the event handler twice
+  fireEvent.pointerDown(button)
+  fireEvent.pointerCancel(button)
+  expect(onPress).toHaveBeenCalledTimes(4)
+  fireEvent.click(button)
+  expect(onPress).toHaveBeenCalledTimes(5)
```

Comment:
Can I get a little more clarity with the desired behavior here?

From the comments on the last two test blocks… I understand that the "proper touch" is what we expect/want to happen (one call to `onPress`) and the "bad touch" is what we don't want to happen (two calls to `onPress`)?

Can you update the comments in this test case to be explicit about what the desired behavior is?

If this "bad touch" order of events is something we wish to fix, can you create an issue and link to it here in the comment?